docs/UserGuide.md

```diff
@@ -3,14 +3,141 @@ layout: page
+    * [List all employees : `listE`](#list-all-employees--liste)
+    * [Locate employees by name: `findE`](#locate-employees-by-name--finde)
+  * [Project Commands](#project-commands)
+    * [Add a new project : `addP`](#add-a-new-project--addp)
+    * [Edit a project : `editP`](#edit-a-project--editp)
+    * [Delete a project: `deleteP`](#delete-a-project--deletep)
+    * [Mark project(s) as completed: `markP`](#mark-project--s--as-completed--markp)
+    * [Mark project(s) as incomplete: `unmarkP`](#mark-project--s--as-incomplete--unmarkp)
+    * [Edit deadline of (a) project(s): `dlP`](#edit-deadline-of--a--project--s---dlp)
+    * [Prioritise projects: `priorityP`](#prioritise-projects--priorityp)
+    * [List all projects: `listP`](#list-all-projects--listp)
+    * [Locate projects by name: `findP`](#locate-projects-by-name--findp)
+  * [Task Commands](#task-commands)
+    * [Add a new task to a project: `addT`](#add-a-new-task-to-a-project--addt)
+    * [Delete a task from a project: `deleteT`](#delete-a-task-from-a-project--deletet)
+    * [Mark task(s) as completed: `markT`](#mark-task--s--as-completed--markt)
+    * [Mark task(s) as incomplete: `unmarkT`](#mark-task--s--as-incomplete--unmarkt)
+    * [Sort tasks by deadline and completion status: `sortT`](#sort-tasks-by-deadline-and-completion-status--sortt)
+  * [Assignment Commands](#assignment-commands)
+    * [Assign employee(s) to a project: `assignP`](#assign-employee--s--to-a-project--assignp)
+    * [Un-assign employee(s) from a project: `unassignP`](#un-assign-employee--s--from-a-project--unassignp)
+    * [Assign an employee to a task: `assignT`](#assign-an-employee-to-a-task--assignt)
+    * [Un-assign an employee from a task: `unassignT`](#un-assign-an-employee-from-a-task--unassignt)
+  * [Data Storage](#data-storage)
+    * [Save the data](#save-the-data)
+    * [Edit the data file](#edit-the-data-file)
+  * [FAQ](#faq)
+  * [Known issues](#known-issues)
+  * [Command summary](#command-summary)
+<!-- TOC -->
+
+--------------------------------------------------------------------------------------------------------------------
+
+# Introduction
+
+## About TaskHub
 TaskHub is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, TaskHub can get your project management tasks done faster than traditional GUI apps.
 
-* Table of Contents
-{:toc}
+## About this User Guide
+
+## Navigating the User Guide
+
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Information**<br>
+
+</div>
+
+<div markdown="block" class="alert alert-warning">
+
+**:warning: Warning**<br>
+
+</div>
+
+<div markdown="block" class="alert alert-danger">
+
+**:no_entry: Danger**<br>
+
+</div>
+
+<div markdown="block" class="alert alert-success">
+
+**:white_check_mark: Information**<br>
```

Comment:
We have 2 icons that have the same name `Information`. Perhaps we should remove one? I prefer keeping the first one.

Reply:
The alert box color for this one is different. I used `Information` as a very generalised placeholder in the meantime while waiting to see if anyone else would actually use this box. It can be renamed more specifically later on, if, it is used.